day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -284,11 +71,16 @@ to another directory:
 +
 ----
 # mkdir -p /etc/origin/node/pods-stopped
+----
++
+----
 # mv /etc/origin/node/pods/etcd.yaml /etc/origin/node/pods-stopped
 ----
 
 . Clear all old data:
 +
+// TODO should we move this before removing, or just remove?
```

Comment:
@geliu2016 / @hexfusion 

Just a question on if it's okay to just remove the contents of this directory instead of moving/backing it up first?

Reply:
backup should exist before any state is deleted

Reply:
@hexfusion I updated to move it instead of remove it, like was in one of the other procedures. Let me know if you have any preferences of name/location. Right now is:

```
# mv /var/lib/etcd /var/lib/etcd.old
```